### ISSUE TITLE
Calculating property in "All" view

### ISSUE THREAD

**Issue by nummi** [emberjs/ember-inspector#1019]
Passing `this` to the model doesn't work when calculating a property.
https://github.com/emberjs/ember-inspector/blob/master/app/templates/components/mixin-details.hbs#L55

Reproduce:

1. Select "All" view
<img width="120" alt="image" src="https://user-images.githubusercontent.com/3692/64387588-6ee0fe00-d00a-11e9-9300-44c3ef8a0f43.png">

2. Find a CP that needs computing:
<img width="140" alt="image" src="https://user-images.githubusercontent.com/3692/64387598-77393900-d00a-11e9-91c2-d12420248ef6.png">

3. :boom:

Assigning to myself since I wrote it :-|

**Comment by rwwagner90** [comment]
@nummi do you have an example app you are trying this with? There have been lots of other changes, so not sure if `model=this` is the problem or something else.

**Comment by nummi** [comment]
I discovered this poking around on the Apple Music bets site. Pretty sure ‘this’ is incorrect because ‘this’ is the ‘mixin-detail’ component.

**Comment by rwwagner90** [comment]
@nummi gotcha, I just know a lot has changed in a bunch of areas, so wasn't sure if this was the culprit.

**Comment by nummi** [comment]
Pretty sure the above line is the culprit. It’s a year old 🤢

**Comment by rwwagner90** [comment]
Do we have tests for this?

**Comment by nummi** [comment]
I don’t think so. Not sure how to get a property in the “not computed” state.
